jevents/jevents.c

```diff
@@ -58,7 +63,9 @@ static const char *json_default_name(char *type)
 			goto out;
 	}
 
-	cache = getenv("XDG_CACHE_HOME");
+	cache = getenv_copy("JEVENTS_CACHEDIR");
```

Comment:
This is the fix for 254: allow `JEVENTS_CACHEDIR` to be specified which overrides the normal sequence if specified.